config/config.go

```diff
@@ -60,18 +62,22 @@ type viperRuleAllowlist struct {
 	StopWords   []string
 }
 
+type viperGlobalAllowlist struct {
+	TargetRules        []string
+	viperRuleAllowlist `mapstructure:",squash"`
```

Comment:
huh, TIL https://pkg.go.dev/github.com/mitchellh/mapstructure#hdr-Embedded_Structs_and_Squashing

Reply:
I went a bit crazy at first trying to get `parseAllowlist` to work.